### DISCUSSION TITLE
ffmpeg audio filter option using -af

### DISCUSSION THREAD

**Discussion by a2nath** [SYSTRAN/faster-whisper#801] [Q&A]
Hi I have to write my own scripts to process some audio recording with some background noise. Usually I use the -af with arnndn flag and the path to the noise rejection model .rnn 

Is it possible to integrate this into the package? Is this something where one of us, community members have to create and create a pull request? 

On a separate note: 

Another idea is to check if the whisper engine sees certainty of words that are below < X% and tries to re-encode/re-decode the audio file with background noise rejection in a feedback loop until the number of attempts parameter is met, or until the threshold X% is crossed.

Something like 
bool noise_reject = (.....).set_default(False)
int    noise_attempts = (......).set_default(10)
float certainty_thr = (.....).set_default(0.75) # output layer nodes e.g. ReLU

**Comment by trungkienbkhn** [comment]
Yes you can open a new PR to implement this.
